crud.py

```diff
@@ -0,0 +1,140 @@
+import csv
+import os.path
+import logger as lg
+
+
+db_file_name = ''
```

Comment:
Тут можно просто использовать аннотацию.
```suggestion
db_file_name: str
```